SPEC.md

```diff
@@ -80,13 +80,13 @@ The fuzz tests aim to buttress the message conformance tests with extra verifica
 |  Message           | Type              | Coverage | Tests                             |
 |--------------------|-------------------|----------|-----------------------------------|
 | Handshake          | HTTP              | ✅       | `C001`, `C002`, `C003`, `C004`    |
-| AgreementVoteTag   | WS data (Tag: AV) | ❌       |                                   |
+| AgreementVoteTag   | WS data (Tag: AV) | 🚧       |                                   |
 | MsgOfInterestTag   | WS data (Tag: MI) | ✅       | `C005`                            |
```

Comment:
Should `C006` also be in this table? What is it reserved for?

Reply:
I wanna keep similar message tests close.
This is what I put in the SPEC.md for that number:

```
### ZG-CONFORMANCE-006

    _Reserved for another MsgOfInterest test_
```
Plus this in the `msgofinterst.rs` file:
```
// TODO(Rqnsom): c005_t2: when the node initiates the connection, send MSG_OF_INTEREST with all messages enabled
// TODO(Rqnsom): c006: send MSG_OF_INTEREST with all messages disabled and expect no messages afterwards
```